python/ray/data/_internal/execution/interfaces.py

```diff
@@ -445,6 +453,17 @@ def incremental_resource_usage(self) -> ExecutionResources:
         """
         return ExecutionResources()
 
+    def notify_resource_usage(
```

Comment:
This is kind of gross, but not sure of a better way to still trigger autoscaling.